src/main/java/com/playmonumenta/structures/managers/RespawningStructure.java

```diff
@@ -342,11 +354,10 @@ public void forcedRespawn(Player player, boolean ignoreDefaultSpawnerCount) {
 						}
 					} else {
 						int minutesLeft = mTicksLeft / (60 * 20);
-						String minutesPlural = (minutesLeft > 1) ? "s" : "";
 						for (Player p : mWorld.getPlayers()) {
 							if (isNearby(p)) {
 								p.sendMessage(ChatColor.RED + "" + ChatColor.BOLD + mName + " will forcibly respawn in "
-										+ minutesLeft + " minute" + minutesPlural + "!");
+										+ minutesLeft + " minutes!");
 							}
 						}
```

Comment:
I think this was still needed? "1 minutes" is clunky

Reply:
According to IntelliJ it's not used, but I was thinking the same thing. It turns out this code path is never reached with less than 2 minutes remaining.

Reply:
Oh, I see, cool. That's interesting that the IDE caught that.